build/lin.sh

```diff
@@ -140,6 +153,13 @@ if [ "$ALL_AT_VERSION_LATEST" = "false" ]; then exit 1; fi
 
 # Download and build dependencies from source
 
+if [ $DARWIN = true ]; then
+  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --no-modify-path -y
+  if [ $DARWIN_ARM = true ]; then
```

Comment:
Variable needs to be quoted before comparison.
```suggestion
  if [ "$DARWIN_ARM" = true ]; then
```